WalletWasabi.Gui/Program.cs

```diff
@@ -65,14 +65,27 @@ private static void Main(string[] args)
 
 		private static async void AppMainAsync(string[] args)
 		{
-			AvalonStudio.Extensibility.Theme.ColorTheme.LoadTheme(AvalonStudio.Extensibility.Theme.ColorTheme.VisualStudioDark);
-			MainWindowViewModel.Instance = new MainWindowViewModel();
+			try
+			{
+				AvalonStudio.Extensibility.Theme.ColorTheme.LoadTheme(AvalonStudio.Extensibility.Theme.ColorTheme.VisualStudioDark);
+				MainWindowViewModel.Instance = new MainWindowViewModel();
 
-			await Global.InitializeNoWalletAsync();
+				await Global.InitializeNoWalletAsync();
 
-			MainWindowViewModel.Instance.Initialize();
+				MainWindowViewModel.Instance.Initialize();
 
-			Dispatcher.UIThread.Post(GC.Collect);
+				Dispatcher.UIThread.Post(GC.Collect);
+			}
+			catch (Exception ex)
```

Comment:
@danwalmsley what is the current status of this? Do we need to call Environment.Exit(1)? What if I just catch the exception and return in case of error?